render.go

```diff
@@ -152,7 +151,7 @@ func (ren *RenderComponent) generateBufferContent() []float32 {
 	red := colorR
 	green := colorG << 8
 	blue := colorB << 16
-	alpha := uint32(transparency*255.0) << 24
+	alpha := uint32(ren.Transparency*255.0) << 24
```

Comment:
Well in fact my idea on this (this is not final) is that I should be able to control transparency level from any system that wants to do so (as a visual feedback to the used, eg  "you cannot place the building here because of a collision ^^")

This leads me to the idea that the transparency level should be sent to the shader as a uniform and removed from the buffer altogether, and then used in the fragment shader to produce the final color during rendering.

If we do so Transparency should certainly stay public. If we don't do this we'll have to recompute the whole buffer again which seems like a waste to me because fragment shaders are here to compute colors for us anyway...

My concern with this is: will we degrade performance a lot doing so?